src/Storage/appsettings.json

```diff
@@ -1,4 +1,11 @@
 {
+  "PostgreSQLSettings": {
+    "WorkspacePath": "Migration",
+    "AdminConnectionString": "Host=localhost;Port=5432;Username=platform_storage_admin;Password={0};Database=storagedb",
+    "ConnectionString": "Host=localhost;Port=5432;Username=platform_storage;Password={0};Database=storagedb;Include Error Detail=True",
+    "StorageDbAdminPwd": "<get-from-key-vault>",
```

Comment:
I would prefer if we started using UserSecrets instead of relying on correct settings in the appsettings files. At a minimum, move the correct value to the appsettings.Developmen.json file.

Reply:
Done